notebook/static/notebook/js/actions.js

```diff
@@ -462,38 +462,23 @@ define(function(require){
                 env.notebook.show_command_palette();
             }
         },
-        'show-all-line-numbers': {
-            help : 'show line numbers in all cells, and persist the setting',
-            handler: function(env) {
-                env.notebook.line_numbers = true;
-            }
-        },
-        'hide-all-line-numbers': {
-            help : 'hide line numbers in all cells, and persist the setting',
-            handler: function(env) {
-                env.notebook.line_numbers = false;
-            }
-        },
         'toggle-all-line-numbers': {
             help : 'toggles line numbers in all cells, and persist the setting',
             icon: 'fa-list-ol',
             handler: function(env) {
                 env.notebook.line_numbers = !env.notebook.line_numbers;
             }
         },
-        'toggle-toolbar':{
-            help: 'hide/show the toolbar',
+        'toggle-header':{
+            help: 'hide/show the header',
             handler : function(env){
-                $('div#maintoolbar').toggle();
```

Comment:
I also moved header/toolbar toggle logic from actions.js to notebook.js to be consistent with the line numbers toggle implementation.